labelbox/orm/query.py

```diff
@@ -41,7 +41,10 @@ def results_query_part(entity):
         entity (type): The entity which needs fetching.
     """
     # Query for fields
-    fields = [field.graphql_name for field in entity.fields()]
+    fields = [
+        field.graphql_name if field.graphql_name != "customMetadata" else
+        "customMetadata { value schemaId }" for field in entity.fields()
+    ]
```

Comment:
This isn't really maintainable. If we are making custom queries for classes, i'd like to move that logic to the DBObject class that requires the custom query. 

I'm think that a terrible alternative way to support this would be to override the field.graphql_name for the customMetadata field to be the "customMetadata { value schemaId }".

Instead of ^^ maybe we can add a new optional attribute to the field called like field.subquery or something. Then in the field definition we can set that as part of the class. In the list comprehension above we will use the subquery if it is set. E.g.

```
class DataRow:
    custom_metadata = Field.List(DataRowMetadataField, "custom_metadata", subquery ="customMetadata { value schemaId }" )
```
and 
```
 fields = [
        field.subquery if field.subquery is not None else field.graphql_name
    ]
```

Reply:
Love this!